GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -284,6 +284,9 @@ public FormBrowse(GitUICommands commands, string filter)
 
         private void LayoutRevisionInfo()
         {
+            // Handle must be created prior to insertion
+            IntPtr h = CommitInfoTabControl.Handle;
```

Comment:
Do we want to do it in the .ctor?
I would also add the link you found to the comment (https://social.msdn.microsoft.com/Forums/windows/en-US/5d10fd0c-1aa6-4092-922e-1fd7af979663/tabpagesinsert-bug?forum=winforms)

Reply:
I added the link. 
This could have been fixed by restructuring in some way to, so the handle is created implicitly but I did not find any way when playing around briefly.
As the bug/limitation occurs for Insert and this the only place we insert tabs, I suggest to keep it here.